### PR TITLE
Refactor helpers so that we use from OpenZeppelin

### PR DESCRIPTION
**Pull request by ferittuncer**
Also replaced zeppelin-solidity with openzeppelin-solidity

### CODE REVIEW

File: contracts/standard/proxy/ExperimentalProxy.sol [hunk 1]
```diff
@@ -40,6 +40,7 @@ contract ExperimentalProxy {
         // Return data
         bytes memory _retData;
 
+        // solium-disable-next-line security/no-inline-assembly
```
Comment (epiqueras) [inline contracts/standard/proxy/ExperimentalProxy.sol]: Note needed.

Comment (epiqueras) [inline contracts/standard/proxy/ExperimentalProxy.sol]: Not needed. Part of config.

File: contracts/standard/proxy/ExperimentalProxy.sol [hunk 2]
```diff
@@ -80,6 +81,7 @@ contract ExperimentalProxy {
         // Call on-chain handler
         handleProxySuccess(msg.sig, _data, _retData);
 
+        // solium-disable-next-line security/no-inline-assembly
```
Comment (epiqueras) [inline contracts/standard/proxy/ExperimentalProxy.sol]: Not needed. Part of config.

File: contracts/standard/proxy/ExperimentalProxy.sol (hunk 1 above)
Comment (ferittuncer) [reply contracts/standard/proxy/ExperimentalProxy.sol]: I get an error from Solium if I don't add them. Maybe using an old `kleros-scripts`?

Comment (epiqueras) [reply contracts/standard/proxy/ExperimentalProxy.sol]: From running `yarn lint` or from your editor integration?

Comment (ferittuncer) [reply contracts/standard/proxy/ExperimentalProxy.sol]: Yeah we are. I'm updating kleros-scripts dependency.

**Pull request merged**